msprime/ancestry.py

```diff
@@ -477,6 +488,17 @@ def simulate(
     :param float length: The length of the simulated region in bases.
         This parameter cannot be used along with ``recombination_map``.
         Defaults to 1 if not specified.
+    :param bool discrete_coordinates: If True, use discrete coordinates
+        in simulations such that recombination breakpoints and mutational
+        sites can only occur at integer positions along the genome.
+        Multiple mutations can occur at the same site.
+        If False (the default) the recombination breakpoints and mutational
+        sites are continuous floating point values. All sites in the
```

Comment:
"and mutational sites" -> "and coordinates of mutational sites"

Reply:
done.